src/ContractFinder.ts

```diff
@@ -0,0 +1,9 @@
+import * as deploymentExport from "../deployments/export.json";
+
+export function getContractArtifact(contractName: string, networkId: number) {
+  try {
+    return deploymentExport[networkId.toString()][0].contracts[contractName];
```

Comment:
what is [0] accessing?

Reply:
this element is within a nested array. we need to extract the 0th element.